change/@fluentui-react-migration-v8-v9-e1f77d05-b3f5-4c83-8d61-03fdf639bf92.json

```
@@ -0,0 +1,7 @@
+{
+  "comment": "publishing migration package",
+  "type": "none",
+  "packageName": "@fluentui/react-migration-v8-v9",
+  "email": "gcox@microsoft.com",
+  "dependentChangeType": "none"
```

Should be `patch` to trigger package bump and publish of package
```suggestion
  "dependentChangeType": "patch"
```